manage.py

```diff
@@ -20,16 +20,16 @@ def coverage_context():
     with cov.collect():
         yield
     cov.save()
-    print(f'Coverage is {cov.report()}%')
-
     covered = cov.report()
-    fail_under = cov.config.get_option('report:fail_under')
-    if covered < fail_under:
-        raise SystemExit(2)
+    print(f'Coverage is {covered}%')
 
     cov.html_report()
     cov.xml_report()
 
+    fail_under = cov.config.get_option('report:fail_under')
+    if covered < fail_under:
+        raise SystemExit(2)
```

Comment:
Let's be explicit about the reason of setting an unsuccessful return code (a non-zero one) in the program.
```suggestion
        print(
            'Erroring out because the coverage level is lower '
            f'than the mandatory of {fail_under !s}%...',
        )
        raise SystemExit(2)
```